spec/validation/checksums.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require "spec_helper"
+require_relative "../../lib/ht_sip_validator/sip/sip"
+require_relative "../../lib/ht_sip_validator/validation/checksums"
```

Comment:
Need better way of requiring the files to be tested.  There's an idiom that can be used in spec_helper, but I haven't the time to look it up right now.